ckanext/validation/utils.py

```diff
@@ -66,6 +74,10 @@ def process_schema_fields(data_dict):
     elif schema_json:
```

Comment:
just confirming, a stripped string will be false on this if statement allowing the removal if it was indeed blank?

Reply:
An empty string [is falsy](https://docs.python.org/3/library/stdtypes.html#truth-value-testing).